### PR TITLE
CX: small agent fixes around skiller simulation

### PR DESCRIPTION
**Pull request by TarikViehmann**
This PR fixes and improves a few things:
 - iadds the workpiece checks to the static estimator that are reasonably fast, before they were treated as pick and put actions, hence took way too long time
 - updates the navgraph estimator to utilize the pose estimation feature
 - fixes a number of crashes due to operation on deleted facts, remove-robot-from-goal-meta is a dangerous helper function that probably should be removed in the future....
 - fixes agent task updates to accommodate for updated action parameters
 - starts the robots before the central agent to ensure a loaded navgraph generator beforehand.

### CODE REVIEW

File: cfg/conf.d/execution-time-estimator.yaml [hunk 1]
```diff
@@ -37,6 +37,14 @@ plugins/execution-time-estimator:
         name: discard
         exec-time: 8.25
       # shelf picks
+      00-check-wp-absent:
```
Comment (snoato) [inline cfg/conf.d/execution-time-estimator.yaml:40]: oooof

File: src/clips-specs/rcll-central/domain.clp [hunk 2]
```diff
@@ -303,7 +303,7 @@
 		(if (and (= (length$ ?r-active) 2) (eq (nth$ 2 ?r-active) active)
 		                                   (eq ?cf:value TRUE))
 		  then
-		    (printout error ?r-active crlf)
+		    (printout t ?r-active crlf)
```
Comment (snoato) [inline src/clips-specs/rcll-central/domain.clp:306]: reduced visibility, I always relied on seeing 3 red lines when starting up

**Review by snoato — CHANGES_REQUESTED** [review]
small fixes requested

File: src/clips-specs/rcll-central/domain.clp (hunk 2 above)
Comment (TarikViehmann) [reply src/clips-specs/rcll-central/domain.clp:306]: too bad for you!

File: src/clips-specs/rcll-central/execution-monitoring.clp [hunk 3]
```diff
@@ -421,6 +422,7 @@
   (if (> ?delta 0.5) then
   	(printout t "Robot " ?robot " made sufficient progress (" ?delta "m) on "  ?action-name  crlf)
 	(modify ?pt (counter 0) (last-pose ?pose) (start-time ?now))
+	(modify ?at (timeout-duration (+ ?at-timeout ?*MOVE-PROGRESS-COUNTER*)))
```
[the remark this answers is not in the document]
Comment (TarikViehmann) [reply src/clips-specs/rcll-central/execution-monitoring.clp]: good catch, ty

**Pull request merged**